Tests/Orso.Arpa.Api.Tests/IntegrationTests/ProjectsControllerTests.cs

```diff
@@ -260,7 +261,12 @@ public async Task Should_Not_Create_Due_To_Missing_Mandatory_Field()
                 .PostAsync(ApiEndpoints.ProjectsController.Post(), BuildStringContent(createDto));
 
             // Assert
-            responseMessage.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+            responseMessage.StatusCode.Should().Be(HttpStatusCode.UnprocessableEntity);
+            ValidationProblemDetails validationProblemDetails = await DeserializeResponseMessageAsync<ValidationProblemDetails>(responseMessage);
+            validationProblemDetails.Title.Should().Be("One or more validation errors occurred.");
+            validationProblemDetails.Type.Should().Be("https://tools.ietf.org/html/rfc4918#section-11.2");
+            validationProblemDetails.Status.Should().Be(422);
+            validationProblemDetails.Errors.Should().BeEquivalentTo(new Dictionary<string, string[]>() { { "Number", new[] { "'Number' darf nicht leer sein." } } });
```

Comment:
Wie ist Mehrsprachigkeit im BE gedacht? Kommt später dazu, oder? 
Bis dahin alles in De?

Reply:
Da ist Georg dran. Ja, bis dahin alles in Englisch.